manifests/metacontroller.yaml

```diff
@@ -1,47 +1,517 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: compositecontrollers.metacontroller.k8s.io
+  annotations:
+    "api-approved.kubernetes.io": "unapproved, request not yet submitted"
```

Comment:
Turns out the `k8s.io` namespace has been reserved for official Kubernetes resources.

More information [here](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190612-crd-group-protection.md#what-to-do-if-you-accidentally-put-an-unapproved-api-in-a-protected-group)